src/App.js

```diff
@@ -39,6 +39,25 @@ function App() {
         </ThemeProvider>
       </div>
     );
+  } else {
```

Comment:
General comment on breakpoints. 
I'm not sure if you've looked into MUI's setup for it?
https://mui.com/material-ui/customization/breakpoints/

This instead of coding in the breakpoint logic, MUI will decide what CSS to use based on the device side
https://mui.com/material-ui/customization/breakpoints/